### DISCUSSION TITLE
How to make a sequence to sequence prediction in GCN LSTM

### DISCUSSION THREAD

**Discussion by scheng1992** [stellargraph/stellargraph#1967] [Q&A]
I am using the GCN_LSTM function in the stellargraph module to make a multidimensional sequence to sequence prediction.

gcn_lstm = GCN_LSTM(
    seq_len=seq_len,
    adj=sensor_dist_adj,
    gc_layer_sizes=[16, 10],
    gc_activations=["relu", "relu"],
    lstm_layer_sizes=[200, 200],
    lstm_activations=["tanh", "tanh"],
)

Through I can change the prediction length (i.e, predict t+n step instead of t+1). It seems the GCN_LSTM did not support a sequence to sequence prediction. The gcn_lstm.outputs here is by default the number of nodes.

Is there a similar setup as return_sequences=True in the LSTM layer?

**Comment by liam-j-simon** [comment]
I have been wondering about this too. Have you found a solution since posting?
